src/Commands/SixMans.py

```diff
@@ -405,6 +405,14 @@ def leaderboard(author: Member, mentions: str, lbChannelId: int, *arg) -> Embed:
         edited_embed = CaptainsRandomHelpEmbed(embed, blueTeam, orangeTeam, blueCap, orangeCap)
         return edited_embed
 
+    elif (len(arg) == 0 and playerMentioned == False and Leaderboard.countLeaderboard() < 6):
```

Comment:
Instead of a separate `elif` I think it would make more sense to collapse them into an `if-else` inside of the `len(arg) == 0 and playerMentioned == False` `elif`.

Reply:
Resolved.